apps/frontend/src/pages/contributors/libs/helpers/get-contributor-columns/get-contributor-columns.helper.tsx

```diff
@@ -37,6 +37,16 @@ const getContributorColumns = (actions: {
 		header: "Projects",
 		size: 300,
 	},
+	{
+		cell: ({ row: { original: contributor } }) =>
+			contributor.hiddenAt ? (
+				<Icon height={18} name="check" width={18} />
+			) : null,
```

Comment:
To render nothing we are using this approach
```suggestion
			) : <></>,
```